src/force_latest_compatible_version.jl

```diff
@@ -0,0 +1,75 @@
+function _force_latest_compatible_version(ctx::Types.Context; target_name = nothing)
+    direct_deps = load_direct_deps(ctx.env)
+    direct_deps_uuids = getproperty.(direct_deps, Ref(:uuid))
+    nonstdlib_direct_deps_uuids = filter(x -> x ∉ keys(stdlibs()), direct_deps_uuids)
+    return _force_latest_compatible_version(ctx, nonstdlib_direct_deps_uuids; target_name)
+end
+
+function _force_latest_compatible_version(ctx::Types.Context, uuid_list::AbstractVector{Base.UUID};
+                                          target_name = nothing)
+    isempty(uuid_list) && return true
+    results = _force_latest_compatible_version.(Ref(ctx), uuid_list; target_name)
+    return all(results)
+end
+
+function _warn_no_compat(has_compat::Bool,
```

Comment:
Just inline this where it is used. The number of lines just to list the arguments is longer than the function body.

Reply:
Done!